src/DurableSDK/DurableTaskHandler.cs

```diff
@@ -206,15 +235,41 @@ private static object GetEventResult(HistoryEvent historyEvent)
             
             if (historyEvent.EventType == HistoryEventType.TaskCompleted)
             {
-                return TypeExtensions.ConvertFromJson(historyEvent.Result);
+                return ConvertFromJson(historyEvent.Result);
             }
             else if (historyEvent.EventType == HistoryEventType.EventRaised)
             {
-                return TypeExtensions.ConvertFromJson(historyEvent.Input);
+                return ConvertFromJson(historyEvent.Input);
             }
             return null;
         }
 
+        public static object ConvertFromJson(string json)
```

Comment:
This used to exist in `TypeExtensions`, creating a tight-dependency with worker utilities. It's been copied here to allow this folder to be readily copied out into an external SDK